spec/integration/step_down_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 describe 'Step down behavior' do
   require_topology :replica_set
-  min_server_fcv '4.2'
 
+=begin Uncomment to shorten the runtime of this test
```

Comment:
If this shortens the runtime of the test, why do we want it disabled by default?

Reply:
I would prefer to have this approach used by default, but the spec as approved describes the simpler approach which takes longer.

Reply:
Ah, okay, I didn't realize that. Can you update this comment to mention that?

Reply:
What would you like this comment to say?

Reply:
Something about how this is only disabled by default per spec requirements; maybe something like:

```ruby
=begin Uncommenting this will reduce the runtime of the test, but the spec currently does not allow us to do it
```

Reply:
Made the suggested change.

Reply:
Thanks!